petclinic/petclinic-interface/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "petclinic-interface"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Should this be 0.3.1?

Reply:
I think the interface is a "breaking change" because it involves a different version of wasmbus-rpc, so actors can't update to the newer interface without updating wasmbus